json-lint/schemas/inventory/adservers/engageya.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$id": "#engageya",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Engageya adserver",
+    "description": "https://github.com/ampproject/amphtml/blob/master/ads/engageya.md",
+    "additionalProperties": false,
+    "required": [
+        "data-widgetIds",
+        "data-websiteId",
+        "data-publisherId"
+    ],
+    "defaultSnippets": [
+        {
+            "label": "Engageya template",
+            "description": "contains required keys",
+            "body": {
+                "type": "engageya",
+                "layout": "responsive",
+                "data-widgetIds": "$1",
+                "data-websiteId": "$2",
+                "data-publisherId": "$3"
+            }
+        }
+    ],
+    "properties": {
+        "type": {
+            "enum": [
+                "engageya"
+            ]
+        },
+        "layout": {
+            "type": "string"
+        },
+        "data-widgetIds": {
```

Comment:
int or parameter ( look at other adservers)
 type = ["intiger", "string"]
pattern = "\\$\{.*\}"     ((something similar))

Ad tests for both cases!  (make one ad server in the engayaa.test.json with params, one with natural values)